pom.xml

```diff
@@ -3,16 +3,23 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>1.580</version>
+    <version>2.2</version>
   </parent>
 
+  <properties>
+    <jenkins.version>1.577</jenkins.version>
+    <java.level>7</java.level>
+    <jenkins-test-harness.version>2.1</jenkins-test-harness.version>
```

Comment:
You should not need it if the code is correct.
IIUC you suppress new injected tests which legitimately fail